aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/convert/cde/DocumentAttributeConditionConverter.java

```diff
@@ -0,0 +1,30 @@
+package software.amazon.kendra.datasource.convert.cde;
+
+import software.amazon.awssdk.services.kendra.model.DocumentAttributeCondition;
+
+public class DocumentAttributeConditionConverter {
+
+  public static software.amazon.awssdk.services.kendra.model.DocumentAttributeCondition toSdkDocumentAttributeCondition(
+      final software.amazon.kendra.datasource.DocumentAttributeCondition documentAttributeCondition) {
+    if (documentAttributeCondition == null) {
+      return null;
+    }
+    return DocumentAttributeCondition.builder()
+        .conditionDocumentAttributeKey(documentAttributeCondition.getConditionDocumentAttributeKey())
+        .operator(software.amazon.awssdk.services.kendra.model.ConditionOperator.fromValue(documentAttributeCondition.getOperator()))
+        .conditionOnValue(DocumentAttributeValueConverter.toSdkDocumentAttributeValue(documentAttributeCondition.getConditionOnValue()))
+        .build();
+  }
+
+  public static software.amazon.kendra.datasource.DocumentAttributeCondition toModelDocumentAttributeCondition(
+      final software.amazon.awssdk.services.kendra.model.DocumentAttributeCondition documentAttributeCondition) {
+    if (documentAttributeCondition == null) {
+      return null;
+    }
+    return software.amazon.kendra.datasource.DocumentAttributeCondition.builder()
+        .conditionDocumentAttributeKey(documentAttributeCondition.conditionDocumentAttributeKey())
+        .operator(documentAttributeCondition.operator().toString())
```

Comment:
if .operator() is null what will happen here?

Reply:
since operator is required for documentAttributeCondition, we should never have stored a documentAttributeCondition with null operator. It seems we don't usually try to catch such anomaly or throw exception in this package. If operator() is null I feel a nullpointer exception actually makes more sense than swallowing the bug.

Reply:
I raise this question because we have had a previous issue in the webcrawler of this exact problem. 
In that case we were missing the null check. From what you're saying operator() is required, and so it cannot be null. LGTM

Ref: https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-kendra/commit/c7396d114753285e221aeaa459672f69ee5ed38c